app/views/providers/allocations/_allocation_report_closed_state.html.erb

```diff
@@ -0,0 +1,32 @@
+<% if allocations.present? %>
+  <% unique_heading_id = rand(1...100) %>
```

Comment:
what is this for?

feels odd

Reply:
accessibility, connecting the table to the h2